### PR TITLE
CS-Script v1.7.8.0 update

### PR DESCRIPTION
**Pull request by oleg-shilo**
This is the update for CS-Script.Npp plugin.

### CODE REVIEW

File: plugins/validate.json [hunk 1]
```diff
@@ -210,9 +210,27 @@
...
+        "fe4cde230841081e808c11a5befc1cdc": "CompatibilityTest.exe",
+        "64b2eaaf32ead7435205034acc49327e": "cscs.exe",
+        "8bf4b1343f572ed212e9df8ea1669ad2": "cscs.v3.5.exe",
+        "2bdf3b878ecd4005afaafea20e628ec0": "css_dbg.exe",
+        "ee81ba12da559a55ee7d0f052faaed32": "csws.exe",
+        "749e914bb5eb3248042ab02d15c01bfe": "launcher.exe",
+        "a631f0551232e7e0512870a4e274f4d5": "npp_jit.exe",
+        "d2b71f5cfae2d0e1b4a8d993c1ef43b8": "nuget.exe",
+        "a6cf760b1b47db3114678d87a843ea01": "syntaxer.exe",
+        "83f4ba7af2689b0eabea10fed9766133": "Updater.exe",
```
Comment (chcg) [inline plugins/validate.json]: seems there is a comma too much here

**Comment by chcg** [comment]
@oleg-shilo Did you intentionally close it ?

**Comment by oleg-shilo** [comment]
Yes I did. I discovered that the list of MD5s was incomplete. So I didn't want you to work on something that was doomed to fail the verification.

Thank you for lightning fast merger.

**Comment by oleg-shilo** [comment]
I have a small question though.

I am guessing that when you are done with consolidating the both x64 and x86 PluginManagers both of them will be using the same repository (or even the same XML) to source their list of plugins.

Is my guess correct?

**Comment by chcg** [comment]
Dave started with https://github.com/bruderstein/npppm3 for consolidation x32 and x64 unicode, but as donho announced the development of his own system, see https://github.com/notepad-plus-plus/notepad-plus-plus/tree/master/PowerEditor/src/WinControls/PluginsAdmin, so the work here is stalled for now, see https://github.com/bruderstein/nppPluginManager/issues/80. So the future is uncertain.

**Comment by oleg-shilo** [comment]
Thanks. Interesting reading. 

Unfortunately the pattern is too common. A big product has some gaps and the vendor refuses to recognize them. The community and enthusiasts do fill the gaps and for years maintain the solution. Then the vendor wakes up and starts its own effort completely disregarding the favor that was taken from the OS community.

I actually have been there and perfectly understand the uncertainty. 

All the best.

**Pull request merged**